flux-common/src/main/java/software/amazon/aws/clients/swf/flux/poller/TaskNaming.java

```diff
@@ -107,9 +107,54 @@ public static String createActivityId(String stepName, long retryAttempt, String
     }
 
     /**
-     * Given a step name, constructs a partition metadata marker name.
+     * Given a step name and information about how many metadata markers there are,
+     * constructs a partition metadata marker name.
```

Comment:
This comment is no longer correct due to you moving some stuff around

Reply:
Fixed.